apps/web/src/pages/farms/auction.tsx

```diff
@@ -1,3 +0,0 @@
-import FarmAuction from '../../views/FarmAuction'
-
-export default FarmAuction
```

Comment:
Hier gab es probleme bei dem build wegen fehlenden Daten. Hab die seite einfach erstmal gelöscht, da es eh ungenutzt ist.

Reply:
das ist top. FarmAuction wird in naher Zukunft auch nicht genutzt werden.